index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/5be9dbaed1.js" crossorigin="anonymous"></script>
+    <title>Portfolio Samuel!</title>
+    
+</head>
+
+<body>
+
+  <div id="main-container">
+    <header>
+        <nav>
+            <div class="mobile">
+                <div class="header-mobile">
+                    <h1>Welcome</h1>
+                    <form action="#" method="get">
+                    <button id="bmenu"><span class="material-symbols-outlined"> menu </span></button>
+                  </form>
+                </div>
+            </div>
+
+        </nav>
+                <div class="introduction">
+                    <h1>Hey there. I`m Samuel glad to see you</h1>
+                    <h2>I`m a sofware developer</h2>
+                    <p>I`m a student in the world of sofware developer, in a few months I be able
+                        to help to build a product, feature or website Look througth some of my work and
+                        experience! if you like what you see and have a project you need coded don`t hestiate to contact me.
+                    </p>
+                    <section class="social-button">
+
+                    <form action="https://www.youtube.com/watch?v=9i0qqA2JLDs&list=RDMM&index=11" method="get">
+                    <button class="social-t"><i class="fa-brands fa-twitter"></i></button>
+                    </form>
+                    <form action="#" method="get">
+                    <button class="social-t"><i class="fa-brands fa-linkedin-in"></i></button>
+                    </form>
+                    <form action="#" method="get">
+                    <button class="social-t"><i class="fa-brands fa-medium"></i></button>
+                    </form>
+                    <button class="social-t"><i class="fa-brands fa-github"></i></button>
+                    <form action="#" method="get">
+                    <button class="social-t"><i class="fa-brands fa-angellist"></i></button>
+                    </form>
+                    </section>
+
+                </div>
+                <div class="down-button">
+                  <form action="#" method="get">
+                    <button id="down-b"><i class="fa-solid fa-chevron-down"></i></button>
+                  </form>
+                </div>
```

Comment:
- [ ] Please include all this code which is the headline section inside a `<section>` tag instead of wrapping the social icons only in the `<section>` tag for semantic purposes.